src/cli.js

```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env node
-var shx = require('./shx');
+import 'babel-polyfill';
```

Comment:
Polyfill needs to be included in the entry point.  We don't yet have bundles so I am treating the cli as the entry point.  This lets us use future native features, like the `includes` method for arrays.